app/main.py

```diff
@@ -1 +1,58 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.extend(skill.split())
```

Comment:
engineer = SoftwareEngineer()
engineer.learn_skill("Django Rest Framework")

What do you expect as a result?
As for me, skill == ["Django Rest Framework"] but your code will say another